static/styles/_base/_tables.scss

```diff
@@ -248,11 +258,11 @@ table {
       .table__row--nested {
         .table__cell {
           width: 20%;
-          padding: 1rem 0;
+          padding: .5em 0;
```

Comment:
Because I wanted it to be relative to the font-size of the text within, but I guess it doesn't matter.